packages/cfd/src/features/Components/proof-of-identity.tsx

```diff
@@ -0,0 +1,49 @@
+// @ts-expect-error remove this line when ProofOfIdentityContainerForMt5 is converted to TS
+import ProofOfIdentityContainerForMt5 from '@deriv/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container-for-mt5.jsx';
+import React from 'react';
+import { useStore, observer } from '@deriv/stores';
+import type { TCoreStores } from '@deriv/stores/types';
+
+type TCFDValue = {
+    poi_state: string;
+};
+
+type TFormValues = {
+    poi_state?: string;
+};
```

Comment:
These as well. Not type safe. Status can be anything in string. Create a constant or take it from api-types. 😸